src/Libraries/RevitNodes/Elements/ImportInstance.cs

```diff
@@ -27,7 +27,22 @@ public override Autodesk.Revit.DB.Element InternalElement
             get { return InternalImportInstance; }
         }
 
-        internal Autodesk.Revit.DB.ImportInstance InternalImportInstance { get; private set; }
+        internal Autodesk.Revit.DB.ImportInstance InternalImportInstance
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Set the internal Element, ElementId, and UniqueId
+        /// </summary>
+        /// <param name="element"></param>
+        private void InternalSetElement(Autodesk.Revit.DB.ImportInstance element)
```

Comment:
@ksobon I don't think this method is necessary. We already have `InternalSetImportInstance`